### PR TITLE
Fix repo url

### PR DESCRIPTION
**Pull request by akira6592**
# What does this PR do?

Now the repo url https://github.com/redhat-cop/controller_configuration is redirected to https://github.com/redhat-cop/infra.aap_configuration.

So I fixed the repo url of `infra.controller_configuration` to  https://github.com/redhat-cop/infra.controller_configuration in docs.

# How should this be tested?

# Is there a relevant Issue open for this?

# Other Relevant info, PRs, etc

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ authors:
     - Tom Page @Tompage1994
     - Sean Sullivan @sean-m-sullivan
     - David Danielsson @djdanielsson
-repository: https://github.com/redhat-cop/controller_configuration/
-issues: https://github.com/redhat-cop/controller_configuration//issues
+repository: https://github.com/redhat-cop/infra.controller_configuration/
+issues: https://github.com/redhat-cop/infra.controller_configuration//issues
```
Comment (djdanielsson) [inline galaxy.yml]: it looks like a double slash at some point got put in here, while you are at it could you fix that?

**Review by djdanielsson — CHANGES_REQUESTED** [review]

File: galaxy.yml (hunk 1 above)
Comment (akira6592) [reply galaxy.yml]: @djdanielsson Thank you for reviewing. I fixed it.

**Review by sean-m-sullivan — APPROVED** [review]

**Comment by sean-m-sullivan** [comment]
Needs updated links are broken, and other pieces need to be fixed

**Comment by akira6592** [comment]
@sean-m-sullivan Do you think should I fix other links by this PR?

e.g.
https://github.com/redhat-cop/infra.controller_configuration/blob/37ccec9187ae77baf66c9011c1818ed0a2abdfa2/roles/dispatch/README.md?plain=1#L116

**Review by djdanielsson — APPROVED** [review]
LGTM

**Comment by djdanielsson** [comment]
Oops didn't see the comments until after I hit merge cuz I saw the change I had wanted was done

**Comment by akira6592** [comment]
If it needed, I will fix other links by an other PR.

**Pull request merged**